types/profile.pp

```diff
@@ -1,15 +1 @@
-type Openscap::Profile = Enum[
-  'xccdf_org.ssgproject.content_profile_test',
-  'xccdf_org.ssgproject.content_profile_CS2',
-  'xccdf_org.ssgproject.content_profile_common',
-  'xccdf_org.ssgproject.content_profile_server',
-  'xccdf_org.ssgproject.content_profile_stig-rhel6-server-upstream',
-  'xccdf_org.ssgproject.content_profile_usgcb-rhel6-server',
-  'xccdf_org.ssgproject.content_profile_rht-ccp',
-  'xccdf_org.ssgproject.content_profile_CSCF-RHEL6-MLS',
-  'xccdf_org.ssgproject.content_profile_C2S',
-  'xccdf_org.ssgproject.content_profile_test',
-  'xccdf_org.ssgproject.content_profile_rht-ccp',
-  'xccdf_org.ssgproject.content_profile_common',
-  'xccdf_org.ssgproject.content_profile_stig-rhel7-server-upstream'
-]
\ No newline at end of file
+type Openscap::Profile = Pattern[/xccdf_[^_]+_profile_.+/]
```

Comment:
We might want to spec this out to ensure it accepts what we expect

Reply:
it's easy now http://rspec-puppet.com/documentation/type_aliases/